apps/src/gamelab/NewItemTile.jsx

```diff
@@ -0,0 +1,71 @@
+/** List item placeholder for adding a new item */
+'use strict';
+
+var color = require('../color');
+var TileButtons = require('./TileButtons.jsx');
+var TileThumbnail = require('./TileThumbnail.jsx');
+
+/**
+ * List item control (usable in sequences or frames lists) for adding
+ * a new item - displays as a plus sign in a dashed box.
+ */
+var NewItemTile = React.createClass({
+  propTypes: {
+    label: React.PropTypes.string.isRequired
+  },
+
+  render: function () {
+    var styles = {
+      tile: {
+        width: '100%',
+        paddingTop: 4,
+        paddingBottom: 4,
+        marginBottom: 4
+      },
+      wrapper: {
+        position: 'relative',
+        display: 'block',
+        paddingTop: '100%'
+      },
+      dottedBorder: {
+        position: 'absolute',
+        top: 0,
+        bottom: 0,
+        left: 0,
+        right: 0,
+        borderRadius: 9,
+        border: 'dashed 4px ' + color.light_purple,
+        textAlign: 'center',
+        paddingTop: '50%'
+      },
+      addButton: {
+        color: color.light_purple,
+        fontSize: 48,
+        marginTop: '-50%'
+      },
+      sequenceName: {
+        marginLeft: 4,
+        marginRight: 4,
+        marginTop: 4,
+        textAlign: 'center',
+        userSelect: 'none',
+        fontWeight: 'bold',
+        fontStyle: 'italic'
+      }
+    };
```

Comment:
Do you have thoughts on defining `styles` inside render vs. having it defined outside of the class? I feel like I've seen both. Having it inside render allows you to have dynamic config here. Having it outside of the class (and extending styles for dynamic config inside render) makes it more clear what aspects are dynamic.